conf/test_domain_classification.config

```
@@ -11,24 +11,22 @@
 */
 
 params {
-    config_profile_name        = 'Test profile for skipping all short read preprocessing'
-    config_profile_description = 'Minimal test dataset to check pipeline function when all short read preprocessing is skipped.'
+    config_profile_name        = 'Test profile for domain classificaiton of bins.'
```

```suggestion
    config_profile_name        = 'Test profile for domain classification of bins.'
```
But isnt that rather `'Test profile for domain classification of bins and skipping all short read preprocessing.'`?

---

Yes, I think that's probably better - thought the latter was less relevant but best to be explicit.